scipy/signal/windows/windows.py

```diff
@@ -2145,9 +2151,8 @@ def get_window(window, Nx, fftbins=True):
         except KeyError as e:
             raise ValueError("Unknown window type.") from e
 
-        params = (Nx,) + args + (sym,)
+        params = (Nx,) + args
     else:
         winfunc = kaiser
-        params = (Nx, beta, sym)
-
-    return winfunc(*params)
+        params = (Nx, beta)
+    return winfunc(*params, sym=sym)
```

Comment:
I think this passing argument process is problematic. If a window function accept second optional argument, and a user does not input it, the `sys` input to the second argument of the window function.

Reply:
If that's true then indeed it sounds like a bug. I would start by using TDD / writing a test that shows that this is a problem, then fixing the code so that it's handled properly